receiver/fluentforwardreceiver/receiver.go

```diff
@@ -29,6 +29,8 @@ import (
 // FluentBit and increase throughput.
 const eventChannelLength = 100
 
+const defaultBindEndpoint = "0.0.0.0:8006"
```

Comment:
Is this a known port number for Forward protocol?

Reply:
Based on the documentation it does not seem to be the default, the default is `5170` https://docs.fluentbit.io/manual/pipeline/inputs/tcp